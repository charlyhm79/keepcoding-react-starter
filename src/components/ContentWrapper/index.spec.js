import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import ContentWrapper from './';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<ContentWrapper />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('Matchs snapshot', () => {
  const tree = renderer.create(<ContentWrapper />).toJSON();
  expect(tree).toMatchSnapshot();
});
