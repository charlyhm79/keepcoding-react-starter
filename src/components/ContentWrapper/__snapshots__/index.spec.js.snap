// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matchs snapshot 1`] = `
<div
  className="sc-bdVaJa dIIIXG"
/>
`;
